Sources/SwifterSwift/SwiftStdlib/IntExtensions.swift

```diff
@@ -154,6 +154,18 @@ public extension Int {
         return number == 0 ? self : Int(round(Double(self) / Double(number))) * number
     }
 
+    /// Iterates from this number down to the given number, inclusive,
+    /// decrementing by one each time.
+    ///
+    /// - Parameters:
+    ///   - to: another Int to go down to
+    ///   - callback: The closure to call
+    func down(to: Int, callback: (_ current: Int) -> Void) {
+        guard to < self else {
```

Comment:
Since you're working with a closed range you could make this check `<=`. Otherwise you can use an open range.